tests/main.tf

```diff
@@ -0,0 +1,127 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "lx-instance-centos6" {
+  source = "git::https://github.com/plus3it/terraform-aws-watchmaker//modules/lx-instance/"
+
+  Name             = "lx-instance-centos6-${local.test_suffix}"
+  AmiId            = "${data.aws_ami.centos6.image_id}"
+  AmiDistro        = "CentOS"
+  AppVolumeSize    = "10"
+  KeyPairName      = "${local.key_name}"
+  NoPublicIp       = "false"
+  SecurityGroupIds = "${aws_security_group.this.id}"
+  SubnetId        = "${data.aws_subnet_ids.public.ids[0]}"
```

Comment:
There is a very good chance we'll want to run tests in a non-default VPC... Maybe add a `subnet_id` var, and use a local to pick between the two.